docarray/proto/docarray.proto

```diff
@@ -46,6 +46,15 @@ message NodeProto {
 
     // a sub DocumentArray
     DocumentArrayProto chunks = 5;
+
+    NdArrayProto embedding = 6;
+
+    string any_url = 7;
```

Comment:
I am not convinced by this. This is going to explode.

Reply:
I believe the meaning of the Document should be provided by the class, not by the Proto

Reply:
Lets do it for now, in the future we might have to encode this type in a schema field in the proto but for now it is not needed